spring-xd-messagebus-spi/src/main/java/org/springframework/xd/dirt/integration/bus/MessageBusSupport.java

```diff
@@ -1039,27 +1035,37 @@ public T lookupSharedChannel(String name) {
 	 */
 	abstract static class JavaClassMimeTypeConversion {
 
+		public static final MimeType APPLICATION_OCTET_STREAM_MIME_TYPE = MimeType.valueOf(APPLICATION_OCTET_STREAM_VALUE);
```

Comment:
I think we can accomplish this with only String values and possibly eliminate the need for the cache.